src/components/fileHandler.ts

```diff
@@ -3,8 +3,8 @@ import fs from 'fs'
 import util from 'util'
 import { pipeline } from 'stream'
 import getAppRootDir from './getAppRoot'
-import { pgClient } from './postgresConnect'
 import * as config from '../config.json'
+import PosgresDB from '../components/postgresConnect'
```

Comment:
Spelling on Import module -- should be `PostgresDB` ?

Reply:
Yes. Thanks!